okapi/services/logs/submit/WebService.php

```diff
@@ -312,11 +312,9 @@ private static function _call(OkapiRequest $request)
             # NOTICE: We are including EXTERNAL OCDE libraries here! This
             # code does not belong to OKAPI!
 
-            if (Settings::get('OC_BRANCH') == 'oc.de')
+            if (Settings::get('OC_BRANCH') === 'oc.de')
             {
-                $opt['rootpath'] = $GLOBALS['rootpath'];
                 $opt['html_purifier'] = Settings::get('OCDE_HTML_PURIFIER_SETTINGS');
-                require_once $GLOBALS['rootpath'] . 'lib2/OcHTMLPurifier.class.php';
 
                 $purifier = new \OcHTMLPurifier($opt);
```

Comment:
**Fatal error:** Class 'OcHTMLPurifier' not found in /mnt/ocde/htdocs/okapi/services/logs/submit/WebService.php on line 319

Note: For testing methods with auth level > 2, you can add a debug username to the options; here in line 29, e.g.

`'min_auth_level' => 3, 'DEBUG_AS_USERNAME' => 'following5'`